changelog.md

```diff
@@ -3,6 +3,14 @@
 ## UNRELEASED
 _Nothing Yet_
 
+
+## 2017-10-16 - Reack Lists and List Items can now have a className attribute added
```

Comment:
Please don't add this line yet, put it under unreleased for now. We'll update this when we publish.

Reply:
Ah, sorry! Never noticed that bit